daemons/attrd/pacemaker-attrd.h

```diff
@@ -127,7 +127,6 @@ enum attrd_attr_flags {
 };
 
 typedef struct attribute_s {
-    char *uuid; /* TODO: Remove if at all possible */
```

Comment:
It's interesting that this TODO has been around as long as the code that added the uuid in the first place.